### PR TITLE
Use percent line-height four Outlook 2007/10/13 (Fix #590)

### LINKED ISSUE #590: Line height not working
**How can we reproduce this bug?**
Just by using it out of the box, inlining it and then testing in Outlook 2007+

Write out the HTML (or Inky code) that causes the issue.

```
body,
table.body,
p,
td,
th {
  font-size: $global-font-size;
  line-height: $body-line-height;
}
```

**What did you expect to happen?**
The line height to be consistent across all email clients

**What happened instead?**
The line height collapsed

**What email clients does this happen in?**
Outlook 2007+

I've tried using `mso-line-height-rule: exactly;` but it doesn't work

### PR DESCRIPTION
**Pull request by jsit**
Fixes #590

### CODE REVIEW

**Comment by xerxesj** [comment]
Nice fix, but I noticed that the percentage based line-height was adding a bunch of bottom padding to all my buttons. If you add a `line-height: 100%;` to both `<a>` and `<td>` tags within `table.button` fixes the issue.

``` css
table.button {
  td,
  a {
    line-height: 100%;
  }
}
```

File: scss/settings/_settings.scss [hunk 1]
```diff
@@ -58,7 +58,7 @@ $global-font-color: $black;
 $body-font-family: Helvetica, Arial, sans-serif;
 $global-font-weight: normal;
 $header-color: inherit;
-$global-line-height: 1.3;
+$global-line-height: 130%;
```
Comment (IamManchanda) [inline scss/settings/_settings.scss]: @jsit Settings file is generated automatically 

Revert from here and change it here => https://github.com/zurb/foundation-emails/blob/develop/scss/components/_typography.scss#L27

File: scss/settings/_settings.scss [hunk 2]
```diff
@@ -74,7 +74,7 @@ $paragraph-margin-bottom: 10px;
 $small-font-size: 80%;
 $small-font-color: $medium-gray;
 $lead-font-size: $global-font-size * 1.25;
-$lead-line-height: 1.6;
+$lead-line-height: 160%;
```
Comment (IamManchanda) [inline scss/settings/_settings.scss]: Revert from here and change it here => https://github.com/zurb/foundation-emails/blob/develop/scss/components/_typography.scss#L91

**Review by IamManchanda — CHANGES_REQUESTED** [review]

**Comment by jsit** [comment]
Fixed, @IamManchanda. Thanks.

**Comment by tdhartwick** [comment]
nice! Thanks!

**Pull request merged**